lldb/test/API/lang/swift/array_bridged_enum/main.swift

```diff
@@ -0,0 +1,5 @@
+let array = [SampleEnum.zero, SampleEnum.one, SampleEnum.two, SampleEnum.three]
+print(array) //%self.expect('frame variable array[0]', substrs=['[0]', '.SampleEnumZero'])
+             //%self.expect('frame variable array[1]', substrs=['[1]', '.SampleEnumOne'])
+             //%self.expect('frame variable array[2]', substrs=['[2]', '.SampleEnumTwo'])
+             //FIXME: frame variable array[3] returns [.SampleEnumOne, .SampleEnumTwo]
```

Comment:
When you find out why this is happening, I'd love to know what the reason is.

Reply:
Do you mean the FIXME? I saw it when I wrote the test, I have no idea why it's happening yet.

Reply:
file a radar for this?

Reply:
Will do